package.json

```diff
@@ -176,7 +176,8 @@
     "micromatch": "^4.0.8",
     "express": "^4.21.0",
     "path-to-regexp@^1.7.0": "^1.9.0",
-    "path-to-regexp@^6.2.0": "^6.3.0"
+    "path-to-regexp@^6.2.0": "^6.3.0",
+    "http-proxy-middleware": "^2.0.7"
```

Comment:
do we still need this if we're also updating it above?

Reply:
ahh sorry, I should have clarified in the PR comment that we also use the pckage. The update above was for our own use but we need the resolution for react-scripts since we can't upgrade that.